### ISSUE TITLE
No loner able to pull up the recently opened files list

### ISSUE THREAD

**Issue by PurpleAzurite** [ahrm/sioyek#54]
The default binding shift-o, as well as no custom bindings, no longer seem to pull up the files list.

**Comment by ahrm** [comment]
I think it is because we changed the default paths, so the previous files are not being found. Do you still not have any recently opened files after opening some files in the new version?

**Comment by PurpleAzurite** [comment]
No I used the on in AUR.

**Comment by ahrm** [comment]
Can you tell me the output of sioyek when running from command line?

**Comment by PurpleAzurite** [comment]
```
default_config_path: /etc/sioyek/prefs.config
default_keys_path: /etc/sioyek/keys.config
user_config_path: [ 0 ] /usr/share/sioyek/prefs_user.config
user_config_path: [ 1 ] /etc/xdg/sioyek/prefs_user.config
user_config_path: [ 2 ] /etc/sioyek/prefs_user.config
user_config_path: [ 3 ] /home/david/.config/sioyek/prefs_user.config
user_keys_path: [ 0 ] /usr/share/sioyek/keys_user.config
user_keys_path: [ 1 ] /etc/xdg/sioyek/keys_user.config
user_keys_path: [ 2 ] /etc/sioyek/keys_user.config
user_keys_path: [ 3 ] /home/david/.config/sioyek/keys_user.config
database_file_path: /home/david/.local/share/sioyek/test.db
local_database_file_path: /home/david/.local/share/sioyek/local.db
global_database_file_path: /home/david/.local/share/sioyek/shared.db
tutorial_path: /usr/share/sioyek/tutorial.pdf
last_opened_file_address_path: /home/david/.local/share/sioyek/last_document_path.txt
shader_path: /usr/share/sioyek/shaders
Creating shared memory block...
Shared memory created: this is the primary application.
Starting IPC server...
IPC server started.
could not create local databaseunable to open database file
SQL Error: config file change listener constructor called1starting index thread ...
figure indexing finished ...
texture: 5

```

**Comment by PurpleAzurite** [comment]
The .db files no longer exist 🤔

**Comment by PurpleAzurite** [comment]
Okay I deleted them myself on accident. How do i get the program to make them again?

**Comment by ahrm** [comment]
The program should automatically create them when they are not found.

**Comment by PurpleAzurite** [comment]
Manually creating the folder solved the issue. So maybe have QDir check for the folder existing and if not make it.

**Comment by ahrm** [comment]
This should be fixed in 88d2730993b016b59364da4ec4fdcbde4e7f86a3 .
